packages/core/types/tests.tsx

```diff
@@ -112,7 +111,7 @@ interface TestTheme1 {
         </span>
         <span
           className={css`
-            color: theme.secondaryColor;
+            color: ${theme.secondaryColor};
```

Comment:
I think this change is correct @Andarist ?

Reply:
Yes 👌